protocol/src/debugProtocol.ts

```diff
@@ -751,6 +751,8 @@ export module DebugProtocol {
 		expression: string;
 		/** Evaluate the expression in the scope of this stack frame. If not specified, the expression is evaluated in the global scope. */
 		frameId?: number;
+		/** The optional value to set the expression to. */
+		value?: string;
```

Comment:
Maybe this is just me, but I feel like this should be a separate request rather than overloading 'evaluate'.

Either way, I think you need a new capability for this one.